sdk/resourcemanager/azure-resourcemanager-authorization/CHANGELOG.md

```diff
@@ -2,13 +2,9 @@
 
 ## 2.11.0-beta.1 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+#### Dependency Updates
 
-### Other Changes
```

Comment:
You still need `### Other Changes` above `#### Dependency Updates`

Also say
```
- Updated `api-version` of authentication to `2020-08-01-preview`.
```
(as you have msgraph remains same).